docs/methods/NamedAlleleMatcher-201.md

```diff
@@ -74,7 +74,7 @@ PharmCAT's syntax for combination calls uses square brackets to reflect that it
 distinguish it from gene duplications (e.g. tandem arrangements like CYP2D6 `*36+*10`).
 
 A partial allele is when a sample matches all the (core) variants of a defined allele but also has additional variants.
-For example, CYP2C19 `"*2/[*17 + g.94781859G>A]"`.  In the case where a partial call occurs off the reference allele,
+For example, CYP2C19 `*2/[*17 + g.94781859G>A]`.  In the case where a partial call occurs off the reference allele,
```

Comment:
For this, it may be best to show CYP2C19 *2/[*17 + g.94781859G>A] on its own line like:

```
CYP2C19 *2/[*17 + g.94781859G>A]
```

It's being used as an example so calling it out in isolation is more illustrative. Same with the next example of "*2/g.94781859G>A".

Reply:
Inline code blocks are displayed with different background color, so keeping it inline is probably okay?

[docs/using/Research-Mode.md](https://github.com/PharmGKB/PharmCAT/blob/development/docs/using/Research-Mode.md) currently also has these examples inline.